cmake/projects/glbinding/hunter.cmake

```diff
@@ -9,6 +9,17 @@ include(hunter_cmake_args)
 
 # List of versions here...
 
+hunter_add_version(
+    PACKAGE_NAME
+    glbinding
+    VERSION
+    "3.1.0-p0"
+    URL
```

Comment:
This is still the upstream release - you still need to make changes to the fork right?

Reply:
@tnixeu *ping*

Reply:
It should point to the patched fork of glbinding.
I have made a pull request here: https://github.com/cpp-pm/glbinding/pull/1

Reply:
Released here https://github.com/cpp-pm/glbinding/releases/tag/hunter-v3.1.0

Reply:
@tnixeu Please use this release instead: https://github.com/cpp-pm/glbinding/releases/tag/v3.1.0-p0

Reply:
This PR now uses the cpp-pm fork. Thx for forking and merging.